content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -73,22 +75,26 @@ The following `Service` is used for DNS lookups between Cassandra Pods and clien
 1. Launch a terminal window in the directory you downloaded the manifest files.
 1. Create a `Service` to track all Cassandra StatefulSet Nodes from the `cassandra-service.yaml` file:
 
-      ```shell
-      kubectl create -f https://k8s.io/examples/application/cassandra/cassandra-service.yaml
-      ```
+    ```bash
+    kubectl create -f https://k8s.io/examples/application/cassandra/cassandra-service.yaml
+    ```
 
 ### Validating (optional)
 
 Get the Cassandra `Service`.
 
-    kubectl get svc cassandra
+```bash
+kubectl get svc cassandra
+```
 
 The response should be
 
-    NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
-    cassandra   None         <none>        9042/TCP   45s
+```
+NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+cassandra   None         <none>        9042/TCP   45s
+```
 
-If anything else returns, the service was not successfully created. Read [Debug Services](/docs/tasks/debug-application-cluster/debug-service/) for common issues.
+If anything else is returned, the Service was not successfully created. Read [Debug Services](/docs/tasks/debug-application-cluster/debug-service/) for common issues.
```

Comment:
"service creation failed" is more succinct than "the service was not successfully created" and more active.